client/product_details/DefaultView.jsx

```diff
@@ -3,16 +3,27 @@ import React from 'react';
 class DefaultView extends React.Component {
   constructor(props) {
     super(props);
+    
     this.state = {
-      currPhoto: '',
-      photoGallery: []
+      currPhotoIndex: 0,
     };
   }
 
   render() {
-    return (
-      <div>carousel will go here</div>
-    );
+    if (this.props.selectedStyle !== undefined) {
+      var photoGallery = this.props.selectedStyle.photos;
+      return (
+        <div id="defaultView">
+          <a target="_blank" href={photoGallery[this.state.currPhotoIndex].url}>
+            <img src={photoGallery[this.state.currPhotoIndex].url} alt={this.props.selectedStyle.name} className="defaultView" id={this.props.selectedStyle.style_id}></img>
+          </a>
+        </div>
+      );
+    } else {
```

Comment:
I don't think you need the else here becuase of the return statement in the above if block. But that' s also nit-picky. Everything is looking really great.